### PR TITLE
fix: fixing accessibility and visual issues

### PR DESCRIPTION
**Pull request by jsimplicio**

### CODE REVIEW

**Comment by jsimplicio** [comment]
WIP fixes of some of the stuff @Charlotteis pointed out.

File: www/assets/app.css [hunk 1]
```diff
@@ -55,26 +54,42 @@ table {
...
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  body > header nav {
+    position: absolute;
+    right: 0;
+    top: 50%;
+    transform: translateY(-50%);
```
Comment (nickcolley) [inline www/assets/app.css]: is there a way to do this with floats? It'd be better for backwards compatibility.

Comment (jsimplicio) [inline www/assets/app.css]: It's really hard to align them perfectly with floats. I tried everything, even inline-block with vertical align. This was the only way I could align them perfectly. If you can figure out how to align them in another way (check Charlottes issue, it's for accessibility) then please make a PR to help me. Thanks!

Comment (nickcolley) [inline www/assets/app.css]: Having trouble getting the page to load (it can't find hoodie), will try to look soon though sorry.

Comment (jsimplicio) [inline www/assets/app.css]: Also if you're talking about the `transform`, the code still works without it, they just align at a different height.

Comment (nickcolley) [inline www/assets/app.css]: :+1: sorry for the disruption haha

**Comment by nickcolley** [comment]
LGTM :+1:

**Comment by gr2m** [comment]
LGTM :+1:

**Pull request merged**